docs/content/quickstart/mysql-postgres-tutorial.md

```diff
@@ -0,0 +1,318 @@
+      price DECIMAL(10, 5) NOT NULL,
+      product_id INTEGER NOT NULL,
+      order_status BOOLEAN NOT NULL -- Whether order has been placed
+    ) AUTO_INCREMENT = 10001;
+    
+    INSERT INTO orders
+    VALUES (default, '2020-07-30 10:08:22', 'Jark', 50.50, 102, false),
+           (default, '2020-07-30 10:11:09', 'Sally', 15.00, 105, false),
+           (default, '2020-07-30 12:00:30', 'Edward', 25.25, 106, false);
+    ```
+#### Preparing data in Postgres
+1. Enter postgres's container:
+    ```shell
+    docker-compose exec postgres psql -h localhost -U postgres
+    ```
+2. Create tables and populate data
+    ```sql
+   -- PG
+   CREATE TABLE shipments (
+      shipment_id SERIAL NOT NULL PRIMARY KEY,
+      order_id SERIAL NOT NULL,
+      origin VARCHAR(255) NOT NULL,
+      destination VARCHAR(255) NOT NULL,
+      is_arrived BOOLEAN NOT NULL
+    );
+    ALTER SEQUENCE public.shipments_shipment_id_seq RESTART WITH 1001;
+    ALTER TABLE public.shipments REPLICA IDENTITY FULL;
+    INSERT INTO shipments
+    VALUES (default,10001,'Beijing','Shanghai',false),
+           (default,10002,'Hangzhou','Shanghai',false),
+           (default,10003,'Shanghai','Hangzhou',false);
+    ```
+   
+## Starting Flink cluster and Flink SQL CLI
+
+1. Use the following command to change to the Flink directory:
+    ```
+    cd flink-1.13.2
+    ```
+   
+2. Use the following command to start a Flink cluster:
+    ```shell
+    ./bin/start-cluster.sh
+    ```
+   Then we can visit [http://localhost:8081/](http://localhost:8081/) to see if Flink is running normally, and the web page looks like:
+
+   ![Flink UI](/_static/fig/mysql-postgress-tutorial/flink-ui.png "Flink UI")
+
+3. Use the following command to start a Flink SQL CLI:
+    ```shell
+    ./bin/sql-client.sh
+    ```
+   We should see the welcome screen of the CLI client.
+
+   ![Flink SQL Client](/_static/fig/mysql-postgress-tutorial/flink-sql-client.png "Flink SQL Client")
+
+## Creating tables using Flink DDL in Flink SQL CLI
+First, enable checkpoints every 3 seconds
+```sql
+-- Flink SQL                   
+Flink SQL> SET execution.checkpointing.interval = 3s;
+```
+
+Then, create tables that capture the change data from the corresponding database tables.
+```sql
+-- Flink SQL
+Flink SQL> CREATE TABLE products (
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+  ) WITH (
+    'connector' = 'mysql-cdc',
+    'hostname' = 'localhost',
+    'port' = '3306',
+    'username' = 'root',
+    'password' = '123456',
+    'database-name' = 'mydb',
+    'table-name' = 'products'
+  );
+
+Flink SQL> CREATE TABLE orders (
+   order_id INT,
+   order_date TIMESTAMP(0),
+   customer_name STRING,
+   price DECIMAL(10, 5),
+   product_id INT,
+   order_status BOOLEAN,
+   PRIMARY KEY (order_id) NOT ENFORCED
+ ) WITH (
+   'connector' = 'mysql-cdc',
+   'hostname' = 'localhost',
+   'port' = '3306',
+   'username' = 'root',
+   'password' = '123456',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)